test/unit/markdown-it-icons.test.js

```diff
@@ -0,0 +1,35 @@
+const markdownIt = require('markdown-it')()
+  .use(require('../../src/lib/markbind/src/lib/markdown-it/markdown-it-icons'));
+
+test('markdown-it-icons renders icon syntax correctly', async () => {
```

Comment:
Correct me if i'm wrong but I don't think these functions have to be `async`

Reply:
Yeah you are right, thanks for catching this! Have removed the `async`.

Reply:
The rest of the test cases in `parser.test.js` and `Site.test.js` are also using `async`, seems like we can clean those up in a separate PR.